test/run_test.py

```diff
@@ -394,6 +394,11 @@
     'test_jit_fuser_legacy',
 ]
 
+DISTRIBUTED_TESTS = [
```

Comment:
what about some of the other tests such as:

```
'distributed/test_c10d_common',
    'distributed/test_c10d_gloo',
    'distributed/test_c10d_nccl',
    'distributed/test_jit_c10d',
    'distributed/test_c10d_spawn_gloo',
    'distributed/test_c10d_spawn_nccl',
    'distributed/test_store',
    'distributed/test_pg_wrapper',
    'distributed/algorithms/test_join',
```

will they be added to this list later? As well as the RPC tests - we can probably include running them separately from other CI as part of this as well?

Reply:
good point. the target determinator is run on expensive test cases only. 
Since it takes ~3mins to finish walking over all python dependences for all tests listed in TARGET_DET_LIST, it won't be economically reasonable to TD a test that only runs for less than, say 10 seconds.

Reply:
but that being said. if any of these files runs more than 10 sec on avg. definitely let me know and I will add them in later PR.